From: Ben Hutchings <ben@decadent.org.uk>
Subject: bridge: Avoid ABI change in 2.6.32.44

The change is in a private structure defined outside include/.  Hide
the change from genksyms.

diff --git a/net/bridge/br_private.h b/net/bridge/br_private.h
index 8567d47..476b65b 100644
--- a/net/bridge/br_private.h
+++ b/net/bridge/br_private.h
@@ -75,7 +75,10 @@ struct net_bridge_port
 	bridge_id			designated_bridge;
 	u32				path_cost;
 	u32				designated_cost;
+	/* This structure is private; ignore changes */
+#ifndef __GENKSYMS__
 	unsigned long			designated_age;
+#endif
 
 	struct timer_list		forward_delay_timer;
 	struct timer_list		hold_timer;
